published/006-2016-March-7.md

```diff
@@ -0,0 +1,180 @@
+
+```
+> git clone https://github.com/ipfs/station.git && cd station
+> npm install
+> npm start
+```
+
+#### [**website**](https://github.com/ipfs/website)
+
+The list of [API Commands](https://ipfs.io/docs/commands/) on the website has been updated. This provides a single place to look through all of the CLI commands for go-ipfs, at once; it is a good reference point if you're not sure exactly which command to use next, and `ipfs commands` seems sparse.
+
+#### **AUR**
+
+[@Kubuxu](https://github.com/Kubuxu) has made sure that `gx`, `gx-go`, and `ipget` are available on [AUR](https://aur.archlinux.org), the arch-linux user repository. This means that it will be easier to get and install these on linux platforms.
+
+#### [**karma-peer**](https://github.com/dignifiedquire/karma-peer)
+
+[@dignifiedquire](https://github.com/dignifiedquire)'s work on [karma-peer](https://github.com/dignifiedquire/karma-peer) means that it now has the ability to dynamically launch browsers, which will help [@diasdavid](https://github.com/diasdavid) (and hopefully more people!) write better tests for P2P browser applications. See an example of some tests, [here](https://github.com/dignifiedquire/karma-peer/blob/master/test/index.spec.js). You can also read [the discussion](https://github.com/diasdavid/piri-piri#why-does-it-exist) that lead to this module and the original tool to test P2P browser applications.
+
+#### [**randor**](http:https://github.com/dignifiedquire/randor)
+
+[@dignifiedquire](https://github.com/dignifiedquire) has been working on [randor](http:https://github.com/dignifiedquire/randor), a testing framework that will be able to send huge files and lots of requests into IPFS in order to test how it works for edge cases and how it scales. Randor is now able to rerun tests predictably based on stored data, so it's easy to find and fix bugs. [@whyrusleeping](https://github.com/whyrusleeping) is already working on the first bug that randor detected. To contribute, check out the repository.
+
+#### [**WebRTC Explorer**](https://github.com/diasdavid/webrtc-explorer)
+
+WebRTC Explorer 2.0.0 has been alpha released! WebRTC Explorer is essentially a way to bring Packet Switching to the Application Level, using WebRTC Data Channels as the transportation layer between nodes. WebRTC enables communication between browsers without needing mediators (servers), enabling users to route packets between machines, using only Web technologies. WebRTC is inspired by the Chord routing scheme to create routing tables that are evenly balanced across nodes. [@diasdavid](https://github.com/diasdavid) wrote up a more extensive [blog post here](http://blog.daviddias.me/2016/02/28/webrtc-explorer-2-0-0-alpha-release), and has an introductory video [here](https://youtu.be/fNQGGGE__zI).
```

Comment:
- @diasdavid i think "WebRTC Explorer" is more than "a way to bring Packet Switching to the Application Level" no?
- "WebRTC is inspired by the Chord routing scheme to create..." you mean "WebRTC Explorer". WebRTC alone has nothing to do with Chord.